include/libzfs.h

```diff
@@ -500,7 +503,7 @@ _LIBZFS_H void zpool_obj_to_path(zpool_handle_t *, uint64_t, uint64_t, char *,
     size_t);
 _LIBZFS_H int zfs_ioctl(libzfs_handle_t *, int, struct zfs_cmd *);
 _LIBZFS_H void zpool_explain_recover(libzfs_handle_t *, const char *, int,
-    nvlist_t *);
+    nvlist_t *, char *, size_t);
```

Comment:
Is there any concern here (and elsewhere) around breaking already-published ABI?

(I have no strong opinion; I just remember one of my early career tasks was #14597, where not breaking the ABI was the whole point).

Reply:
That is a good point, I had not thought about originally. If more of us think using with older kernel might produce some issue, I can definitely update and write newer versions of these functions.

Reply:
Within a release branch series (e.g. 2.2.x) we go out of our way to avoid breaking the ABI.  This new functionality isn't likely to ever get backported to 2.2.x so in this case it should be already.